docs/stories/bitcoin-integration.md

```diff
@@ -0,0 +1,109 @@
+
+![](../assets/add-custom-output-ln-protocol.png)
+
+Contrary to the other types of output, the coins that make up the new custom output can come from _both_ parties.
+This is a necessary requirement for DLCs and other interesting protocols, where both parties lock up coins in a shared output.
+
+The 10101 wallet application that uses will therefore have to handle all the DLC-specific transaction building and signing, using the shared custom output as the DLC.
+
+### Settling your CFD into the Lightning channel
+
+One huge advantage of the 10101 approach to CFD trading versus what we previously built with ItchySats is that the CFD can be settled directly back into a Lightning channel!
+This means that the off-chain state doesn't have to be committed on-chain and that the coins are readily available to make payments or do whatever else 10101 lets you do in the future.
+
+![](../assets/remove-custom-output-ln-protocol.png)
+
+This protocol is analogous to how a HTLC would be removed from a commitment transaction after a payment is claimed.
+Except for the fact that the coins in the custom output can be _split up_ between the two parties.
+
+### Force closing a channel containing a custom output
+
+There should not be anything inherently different about force closing a Lightning channel containing a custom output.
+The commitment transaction will be published on-chain and all the outputs will have different spend conditions.
+
+The effect on the custom output will be different, as the `rust-lightning` library will not ensure that the output is split among the two parties Bitcoin wallets.
+It will be the responsibility of the application built on top of `rust-lightning` to spend the custom output.
+For the 10101 wallet that will mean publishing a CET according to what the oracle attests.
+
+If you want to learn more about how a CFD protocol using DLCs works in the context of ItchySats, earlier in the year we wrote a [blog post](https://comit.network/blog/2022/01/11/cfd-protocol-explained/) about it.
+It should still be relevant to what we are building today.
+
+## Compatibility with other Lightning nodes
+
+The point of building 10101 on Lightning is to become part of the Lightning Network.
+We are excited to cater to its large and constantly growing userbase.
+To this end, it is very important for us to build a Lightning wallet which is compatible with the rest of the network.
+
+Our modifications to `rust-lightning` should just extend the capabilities of the 10101 Lightning node, without compromising its ability to route payments to and from any other node.
+As such, the 10101 wallet will only be able to do CFD trading with compatible nodes, but it should be able to pay for your coffee at your favourite cafe.
+
+## Reduced scope for the tournament
```

Comment:
Instead of formulating this negatively we could turn it into a positive statement like.

```suggestion
## Future work beyond the Legends of Lightning Tournament
```

Reply:
Much better, thanks!